setup.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
```

Comment:
maybe rewrite to bash, not python?
you can iterate through lines of the file easily:

```bash
cat plugins.txt | while read line || [[ -n $line ]];
do
   # do something with $line here
done
```

Reply:
I tried but it didn't work